media/src/App.svelte

```diff
@@ -477,19 +478,21 @@
             }
         } else if (data.message.pollResponse && isHost) {
             const sessionKey = data.message.session_key;
-            if (data.message.poll === 'select point') {
-                if (pollResponses[sessionKey]) {
-                    objectResponses.remove(pollResponses[sessionKey]);
+            if (!(lockPoll && pollResponses[sessionKey])) {
```

Comment:
Ah okay, makes sense. This is working well from my testing, so I will merge this.

Just as a side note: This kind of intricate logic definitely lends itself to some unit testing, which we haven't been the most adamant about on this project. If possible, I would love to put some tests together for `handleSocketMessage`. This may involve moving it from App.svelte into a utils file, and parameterizing any of the state updates that this function facilitates.